### DISCUSSION TITLE
Will this ever include regular expressions?

### DISCUSSION THREAD

**Discussion by HeroJeanPierre** [OlegLustenko/obsidian-bulk-rename#5] [Ideas]
Would be awesome if this including regular expressions at some point.

**Comment by OlegLustenko** [comment]
How do you want to use RegExp? For Search?

We have RegExp for **Existing Characters**

It's not documented well, but we have an example here: 

 https://youtu.be/0SR6oJZAdEI

**Comment by OlegLustenko** [comment]
I'll add search files by RegExp in the next version. 

I'll take it as highest priority

**Reply by OlegLustenko** [reply]
Shipped!
@HeroJeanPierre Any feedback?

Here is much more possiblities now. 
I'm dreaming if it makes sense to add them too
